sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2669,6 +2669,25 @@ object SQLConf {
       .checkValue(_ >= 0, "The value must be non-negative.")
       .createWithDefault(8)
 
+  val NULL_AWARE_ANTI_JOIN_OPTIMIZE_ENABLED =
+    buildConf("spark.sql.nullAware.antiJoin.optimize.enabled")
```

Comment:
If we don't have other nullAware.xxx, maybe nullAware.antiJoin => nullAwareAntiJoin?

Reply:
done